pkg/controller/common/config.go

```diff
@@ -21,7 +21,8 @@ type config struct {
 
 // OLM is intermediate struct for serialization
 type olm struct {
-	Images images `json:"relatedImage,omitempty"`
+	Images                    images `json:"relatedImage,omitempty"`
+	WebhookManagementDisabled bool   `json:"webhookManagementDisabled,omitempty"`
```

Comment:
Any reason why this is `webhookManagementDisabled` and not `webhookManagementEnabled` that defaults to `true`?

Reply:
I think we went this direction as this will be BW compatible even when the annotation is completely missing (i.e. operator is deployed with old manifest).

Reply:
> 
> 
> I think we went this direction as this will be BW compatible even when the annotation is completely missing (i.e. operator is deployed with old manifest).

Yes, that was exactly the reason.

Reply:
I agree that `webhookManagementEnabled` makes more sense. You can default it to true by using `viper.SetDefault()` afaik

Reply:
I think this should probably be moved into `config.Controller`.